kolibri/core/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
```

Comment:
@rtibbles you're right! I was under the impression that Django reuses database connections (like other DB frameworks), but that's not the case: https://docs.djangoproject.com/en/1.9/ref/databases/#persistent-connections

For testing, I'll set `CONN_MAX_AGE` for now and then start benchmarking again.